src/aero_model.F90

```diff
@@ -604,6 +602,81 @@ end subroutine aero_model_emissions
   ! private methods
   !=============================================================================
 
+  subroutine surf_area_dens( ncol, mmr, pmid, temp, beglev, endlev, sad, reff, sfc )
+    ! djlo : diam kept out as argument
+    use mo_constants     , only : pi
+
+    ! dummy args
+    integer,  intent(in)  :: ncol
+    real(r8), intent(in)  :: mmr(:,:,:)
+    real(r8), intent(in)  :: pmid(:,:)
+    real(r8), intent(in)  :: temp(:,:)
+    ! real(r8), intent(in)  :: diam(:,:,:) ! djlo : kept out as argument
+    integer,  intent(in)  :: beglev(:)
+    integer,  intent(in)  :: endlev(:)
+    real(r8), intent(out) :: sad(:,:)
+    real(r8), intent(out) :: reff(:,:)
+    real(r8),optional, intent(out) :: sfc(:,:,:)
+
+    ! local vars
+    !HAVE TO GET RID OF THIS MODE 0!! MESSES UP EVERYTHING!!
+    real(r8)         :: numberConcentration(pcols,pver,0:nmodes_oslo)
+    real(r8), target :: sad_mode(pcols,pver, nmodes_oslo)
+    real(r8)         :: vol_mode(pcols,pver, nmodes_oslo) ! djlo
+    real(r8)         :: vol(pcols,pver)                   ! djlo
```

Comment:
Are these comments needed? Something descriptive would be nice.

Reply:
I have now removed the comments.